securitas/templates/user.html

```diff
@@ -1,33 +1,19 @@
-{% extends "with_navbar.html" %}
+{% extends "main.html" %}
 
 {% block title %}User: {{ user.username }}{% endblock %}
 
-{% block head %}
-  {{ super() }}
-  {{ css("user") }}
-{% endblock %}
-
 {% block content %}
   {{ super() }}
-  <div class="container section">
-
+  <div class="container">
     <div class="row">
-      <div class="col s12 l3">
-        <h3 class="header light-blue-text text-darken-4 fw300">
-          {{ user.name if user.name else user.username }}
-        </h3>
-        <div class="card">
-          <div class="card-image">
-            <img src="{{ gravatar(user.mail if user.mail else 'default', 400) }}">
-          </div>
-          <div class="card-stacked">
-            <div class="card-content">
-              <span class="card-title">{{ user.username }}</span>
-              {% if user.username == current_user.username %}
-                <p>
-                  <a href="{{ url_for('user_edit', username=user.username) }}">Edit profile</a>
-                </p>
-              {% endif %}
+      <div class="col-4">
+        <h4 class="">{{ user.username }}</h4>
+        {{ user.name if user.name else user.username }}
+        <img class="img-thumbnail" src="{{ gravatar(user.mail if user.mail else 'default', 400) }}">
+        {% if user.username == current_user.username %}
+          <a href="{{ url_for('user_edit', username=user.username) }}" class="btn btn-outline-primary btn-block mb-2">Edit profile</a>
+        {% endif %}
+        <div>
               {% if user.timezone %}
                 <p><b>Timezone:</b> {{ user.timezone }}</p>
```

Comment:
Maybe make these sidebar attributes not be `<p>` now? Because there's quite a bit of space between them in bootstrap.